crest/Assets/Crest/Crest/Scripts/Helpers/Attributes/MultiPropertyAttribute.cs

```diff
@@ -263,4 +263,21 @@ public FilterEnumAttribute(string property, FilteredAttribute.Mode mode, params
             _property = property;
         }
     }
+
+    public class HeadingAttribute : DecoratorAttribute
```

Comment:
could you add bit of documentation for this class. is Heading to enable it to be hideable, or to fix spacing, or both? wonder if the name could be descriptive of what it does, but thats not a critical point, Heading works.

Reply:
Done. Just for hiding. Otherwise it is drop-in replacement for Header.